store/sql_store_test.go

```diff
@@ -0,0 +1,102 @@
+package store
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/go-gorp/gorp"
+	"github.com/go-sql-driver/mysql"
+	"github.com/mattermost/mattermost-mattermod/model"
+)
+
+const (
+	defaultMysqlDSN     = "mattermod:mattermod@tcp(localhost:3306)/mattermost_mattermod_test?charset=utf8mb4,utf8\u0026readTimeout=30s\u0026writeTimeout=30s\u0026parseTime=true"
+	defaultMysqlUser    = "mattermod"
+	defaultMysqlRootPWD = "mattermod"
+	defaultMysqlUserPWD = "mattermod"
+	defaultMysqlDB      = "mattermod_test"
+)
+
+func getTestSQLStore(t *testing.T) *SQLStore {
+	t.Helper()
+
+	createTempDB(t, defaultMysqlDB, getEnv("MYSQL_USER", defaultMysqlUser))
+	t.Log("created temporary database")
+
+	cfg, err := mysql.ParseDSN(defaultMysqlDSN)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cfg.User = getEnv("MYSQL_USER", defaultMysqlUser)
+	cfg.Passwd = getEnv("MYSQL_PASSWORD", defaultMysqlUserPWD)
+	cfg.DBName = defaultMysqlDB
+
+	db, err := sql.Open("mysql", cfg.FormatDSN())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	store := &SQLStore{
+		master: &gorp.DbMap{
+			Db:            db,
+			TypeConverter: mattermConverter{},
```

Comment:
I guess then https://mattermost.atlassian.net/browse/MM-26710 is not required anymore?

Reply:
Yes it looks like the custom scanner is required for now.